### PR TITLE
Havok tutorial for bowling scene

### PR DESCRIPTION
**Pull request by aprilgittens**
Cloud Advocate request from @PirateJC. I created a beginner level tutorial that shows the steps for using Havok Physics Plugin for a simple bowling scene.

### CODE REVIEW

**Review by sebavan — APPROVED** [review]
Really coool !!! thanks a ton for the contrib @aprilspeight

**Review by Popov72 — APPROVED** [review]

File: content/guidedLearning/workshop/Bowling.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+```
+
+Now that physics is enabled, let’s create our bowling lane!
+
+## Create Bowling Lane
+
+The bowling lane is considered the ground in our scene. We want to provide a thin long platform for both the pins to sit and the bowling ball to roll. Using `MeshBuilder`, we create the lane and assign the width and height parameters, 6 and 20 respectively.
+
+```javascript
+var lane = BABYLON.MeshBuilder.CreateGround("lane", { width: 6, height: 20 }, scene);
```
Comment (RaananW) [inline content/guidedLearning/workshop/Bowling.md]: nit - const

**Review by RaananW — COMMENTED** [review]
Hey! this is truly awesome :-)

I just have a small ask, if possible - you are using both var and const, which mixes two types of variable definition paradigms. Will it be possible to change the vars to either const or let? 

If you want me to do that after merging let me know, i'd be happy to.

File: content/guidedLearning/workshop/Bowling.md (hunk 1 above)
Comment (aprilgittens) [reply content/guidedLearning/workshop/Bowling.md]: @RaananW if you could give me until EOD to make the update that'd be perfect. I'd also need to update the Playground that's linked as well. I'll commit my changes and @-mention you once it's ready for your review. Thanks for reviewing the tutorial!

Comment (RaananW) [reply content/guidedLearning/workshop/Bowling.md]: The PR was already approved. The minute you give me the OK i'll merge it. No stress :-)

Comment (aprilgittens) [reply content/guidedLearning/workshop/Bowling.md]: @RaananW could you please check again now to confirm if the changes meet your needs? we don't have such declarations in Python so thanks for your understanding haha!

Comment (sebavan) [reply content/guidedLearning/workshop/Bowling.md]: perfect thanks

**Pull request merged**